ebuild_generator/ebuild.py

```diff
@@ -260,6 +264,8 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
 
         # source configuration
         ret += "src_configure() {\n"
+        if self.name == 'opencv3':
+            ret += "    filter-flags \'-march=*\' \'-mtune=*\'\n"
```

Comment:
It's taken from
https://www.mail-archive.com/gentoo-alt@lists.gentoo.org/msg07234.html